new_course/1_programming_fundementals/basic_enumerable_methods.md

```diff
@@ -298,7 +298,7 @@ Now let's look at a more elaborate example that shows just how powerful this met
 
 Now that you know who's coming to your party, you need to decide where to go. You don't actually like making decisions very much, so you put it to a vote among your friends.
 
-The options are St. Mark's Bistro, a classy place suited for a sophisticated person such as yourself. The other option is Bob's Dirty Burger Shack, which you know its Brian's favorite place. Since he's coming to the party now, it's best to include it as an option to avoid any arguments. Your friends' votes are collected in the `votes` array.
+The options are St. Mark's Bistro, a classy place suited for a sophisticated person such as yourself. The other option is Bob's Dirty Burger Shack, which you know is Brian's favorite place. Since he's coming to the party now, it's best to include it as an option to avoid any arguments. Your friends' votes are collected in the `votes` array.
```

Comment:
This could also be "it's".

Reply:
Nah, "it's" wouldn't be correct here. Your change is good.